doc/releases/release-notes-3.1.rst

```diff
@@ -433,6 +433,11 @@ Drivers and Sensors
 
 * I2C
 
+  * Arbitrary clocking speedd defined in device tree now supported :c:macro:`I2C_SPEED_DT`
```

Comment:
I can't parse this, sorry. What does it mean? Also there is a "speed" typo here.

Reply:
Whoops!